Standards/scs-0001-sovereign-cloud-standards.md

```diff
@@ -0,0 +1,304 @@
+It MUST NOT start with a hyphen and SHOULD start with a lower-case letter.
+It SHOULD NOT contain more than one subsequent hyphen and SHOULD start with a lower-case letter.
+
+The file name of an SCS document is formed using the following pattern:
+`scs-XXXX-vN-T.md`, where
+`XXXX` is replaced with the zero-padded document number,
+`N` is replaced with the major version of the document, and
+`T` is replaced with the slugified title.
+For a document with the number 390, with a major version number 2 and a slugified title `flavor-naming`,
+the resulting file name would be `scs-0390-v2-flavor-naming.md`.
+
+In addition to the number, each document has the following metadata,
+embedded in the markdown header.
+
+| Field name | Requirement | Description |
+| --- | --- | --- |
+| `type` | REQUIRED | one of `Procedural`, `Standard`, or `Decision Record` |
+| `status` | REQUIRED | one of `Draft`, `Stable`, `Deprecated`, or `Rejected` |
+| `track` | REQUIRED | one of `Global`, `IaaS`, `Ops`, `KaaS` |
+| `obsoleted_at` | REQUIRED if `status` is `Deprecated` | ISO formatted date indicating the date after which the deprecation is in effect |
+| `stabilized_at` | REQUIRED if `status` was ever `Stable` | ISO formatted date indicating the date after which the document was considered stable |
+| `rejected_at` | REQUIRED if `status` is `Rejected` | ISO formatted date indicating the date on which the document was rejected |
+| `replaced_by` | RECOMMENDED if `status` is `Deprecated` or `Rejected`, FORBIDDEN otherwise | List of documents which replace this document. |
+| `replaces` | OPTIONAL | List of documents which this document replaces. |
+
+## Sections
+
+Each document SHOULD have the following sections:
+
+- An *Introduction* providing context on the document and linking to other relevant materials.
+- A *Motivation* section which details why this document or the thing it describes is necessary.
+
+In addition, the following OPTIONAL sections should be considered:
+
+- A *Design Considerations* section for Standard type documents,
+  which details other choices
+  which have been considered for the specific feature
+  but were ultimately rejected.
+- An *Open Questions* section which links to issues
+  detailing any open discussion points with respect to a document.
+  This section is RECOMMENDED during the discussion phase (pre 1.0.0)
+  as a "table of contents" of things to work on in that context.
+
+# Process
+
+The lifecycle of an SCS document goes through the following phases:
+Proposal, Draft, Stable, Deprecated, and Rejected.
+
+All decisions for phase transitions follow loose consensus,
+where the group which has to form the consensus depends on the `track` of the document:
+
+- IaaS: The team working on infrastructure-as-a-service topics (formerly Team 1)
+- KaaS: The team working on Kubernetes-as-a-service topics (formerly Team 2)
+- Ops: The team working on operations topics (formerly Team 3)
```

Comment:
Please let's drop the 'formerly Team X'.

Reply:
Ah good suggestion, I was already wondering what to do about IAM.